src/app/shared/capacitor-plugins/mock-app-plugin.ts

```diff
@@ -20,10 +20,47 @@ export class MockAppPlugin implements AppPlugin {
     throw new Error('Method not implemented.');
   }
 
+  // addListener(
```

Comment:
@sultanmyrza Suggest adding a comment about why commenting out and keep these codes so that we can catch up on the reason in the future quickly.